File: Algorithm/Programmers/Lv1/두개뽑아서더하기.py
import collections
def solution(numbers):
    answer = []
    
    for n1 in numbers:
        for n2 in numbers:
            if n1 != n2:
                answer.append(n1+n2)
    
    numbers = Qcollections.Counter(numbers)
    print("collection ",end=' ') 
    print(numbers)
    for key, value in numbers.items():
        if value>=2:
            answer.append(key*2)
    a = set(answer)
    answer = list(a) 
    answer.sort()
    return answer

File: Algorithm/Programmers/Lv1/두개뽑아서더하기.java
import java.util.ArrayList;
import java.util.Arrays;

class Solution {
    public int[] solution(int[] numbers) {
        
        ArrayList<Integer> list = new ArrayList<Integer>();
        for(int i=0; i<numbers.length; i++){
            for(int j=i+1; j<numbers.length; j++){
                if(list.indexOf(numbers[i]+numbers[j])<0){
                    list.add(numbers[i]+numbers[j]);
                }
            }
        }
        
        int[] answer = new int[list.size()];
        for(int i=0; i<list.size(); i++){
            answer[i] = list.get(i);
        }
        Arrays.sort(answer);
        return answer;
    }
}
